### PR TITLE
Add Japanese translated document.

### PR DESCRIPTION
**Pull request by hypermkt**
refs : https://github.com/vuejs/vue-validator/issues/192
### TODO
- [x] Installation
- [x] Getting started
- [x] Validation result structure
- [x] Validator syntax
- [x] v-model integration
- [x] Reset validation results
- [x] Form validatable elements
- [x] Validation classses
- [x] Grouping
- [x] Error messages
- [x] Events
- [x] Lazy initialization
- [x] Custom validator
- [x] Validation timing customization
- [x] Async validation
- [x] API
- [x] Self review
- [ ] Ask for review and fix
### Reference
- [Vue.js 公式サイト日本語翻訳ガイド](https://github.com/vuejs-jp/jp.vuejs.org/blob/lang-ja/CONTRIBUTING.md)

### CODE REVIEW

File: docs/ja/installation.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+### 開発版
+
+    $ git clone https://github.com/vuejs/vue-validator.git node_modules/vue-validator
+    $ cd node_modules/vue-validator
+    $ npm install
+    $ npm run build
+
+CommonJS で使用されるとき、明示的に `Vue.use()` 経由でバリデーターをインストールしなければなりません:
+
+> :warning: もし `vue-router` を使用している場合は、 `router#start` の前に `Vue.use()` でインストールしなければなりません。
```
Comment (hypermkt) [inline docs/ja/installation.md]: When I check on my localhost, I can't see the warning icon. Why...

### current on my localhost

image

### expected

image

Comment (kazupon) [inline docs/ja/installation.md]: You need to install the gitbook plugins.
https://github.com/vuejs/vue-validator/blob/dev/docs/book.json

Comment (hypermkt) [inline docs/ja/installation.md]: Thanks for the comment. I installed the gitbook plugins as follows.

``` sh
$ cd vue-validator/docs
$ gitbook install
$ gitbook build
```

Run to check on localhost.

``` sh
$ gitbook serve
```

`:warning:` was converted into image tag. But I couldn't still see it.
image

The advanced-emoji plugin seems installed successlly on my PC. But the file path on image tag seems different. hmm.

``` sh
$ ll  _book/gitbook/plugins/gitbook-plugin-advanced-emoji/emojis/ | grep warning
-rw-r--r--  1 hypermkt  staff  3173  5  5 21:52 warning.png
```

Comment (hypermkt) [inline docs/ja/installation.md]: I checked the README of this plugin and have no idea... hmm.. Let's move forward the translation first! 
https://plugins.gitbook.com/plugin/advanced-emoji

Comment (kazupon) [inline docs/ja/installation.md]: What's gitbook version ?
If 3.0.0-pre, maybe it might be due not to correspond to the latest version.

**Comment by kazupon** [comment]
/ping @hypermkt 
I fixed en docs.
https://github.com/vuejs/vue-validator/commit/b48759ed2e38309983d63205d17ce9b98d76953b

File: docs/ja/installation.md (hunk 1 above)
Comment (hypermkt) [inline docs/ja/installation.md]: I was using `2.1.3`. (node v6.0.0 installed by nodebrew) At least I can work on translation even if I can't see the icon on localhost. Thanks for helping!

**Comment by hypermkt** [comment]
Thanks for letting me know the update!

File: docs/ja/structure.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      validator: 'customValidator', message: 'invalid fieldX'
+    }]
+  },
+}
+```
+
+様々なトップレベルプロパティはバリデーションスコープの中に、そして各フィールドのバリデーション結果は自身それぞれのスコープの中にあります。
+
+## フィールドバリデーションプロパティ
+- `valid`: フィールドが有効化どうか; もし有効なら、`true` を返し、そうでなければ `false` を返します。
```
Comment (hypermkt) [inline docs/ja/structure.md]: I see the errors by textlink for `;`. 

``` sh
$ textlint structure.md

/Users/hypermkt/work/vue-validator/docs/ja/structure.md
  67:24  error  原則として和文ではセミコロン(;)を使用しません。  preset-jtf-style/4.2.8.セミコロン(;)
  71:31  error  原則として和文ではセミコロン(;)を使用しません。  preset-jtf-style/4.2.8.セミコロン(;)
  72:36  error  原則として和文ではセミコロン(;)を使用しません。  preset-jtf-style/4.2.8.セミコロン(;)
  77:31  error  原則として和文ではセミコロン(;)を使用しません。  preset-jtf-style/4.2.8.セミコロン(;)
```

> ただし、文の途中にハイフン- や セミコン; がある場合は、その記号があると理解しづらい訳になる場合は、例外として削除てもよいです。

Maybe I should delete `;` this time. It doesn't look natural.

File: docs/ja/structure.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+# バリデーション結果構造
+
+バリデーション結果はこの構造でアクセスできます:
+
+```
+{
+  // top-level validation properties
```
Comment (hypermkt) [inline docs/ja/structure.md]: I should probably translate the comments, too.

File: docs/ja/syntax.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+    <div>
+      <span v-if="$validation.id.required">ID is required</span>
+      <span v-if="$validation.id.minlength">Your ID is too short.</span>
+      <span v-if="$validation.id.maxlength">Your ID is too long.</span>
+    </div>
+  </form>
+</validator>
+```
+
+オブジェクトリテラルはルールを与えることができます。上記で示すように、 `required` にはルールが必要なく、 **ダミールール**を代わりに指定することができます。
```
Comment (hypermkt) [inline docs/ja/syntax.md:125]: At first, I couldn't understand the meaning of `dummy rule` in this part. But I could understand when I write a sample code like this `ID: <input type="text" v-validate:id="{ required: we_can_set_here_anything_because_it_is_dummy , minlength: 3, maxlength: 16 }">`.  It works anyway.

File: docs/ja/structure.md (hunk 2 above)
Comment (kazupon) [inline docs/ja/structure.md]: LGTM

File: docs/ja/structure.md (hunk 3 above)
Comment (kazupon) [inline docs/ja/structure.md]: Thank you!

File: docs/ja/syntax.md (hunk 4 above)
Comment (kazupon) [inline docs/ja/syntax.md:125]: Be my guest 😺

File: docs/ja/structure.md (hunk 3 above)
Comment (hypermkt) [inline docs/ja/structure.md]: I'll translate! I got it!

File: docs/ja/structure.md (hunk 2 above)
Comment (hypermkt) [inline docs/ja/structure.md]: I got it!

**Comment by hypermkt** [comment]
Just reached half of docs. Let's keep going! 😄

**Comment by kazupon** [comment]
Oh, You're almost there!  😺

**Comment by hypermkt** [comment]
😄  Let's keep going!

File: docs/ja/timing.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+        <!-- 任意の'initial' は `v-validate` バリデーターに適用されます。(例えば required のみ) -->
+        <input id="password" type="password" v-validate:passowrd="{ required: { rule: true, initial: 'off' }, minlength: 8 }">
+      </div>
+      <input type="submit" value="send" v-if="$validation1.valid">
+    </form>
+  </validator>
+</div>
+```
+
+これは非同期的な特徴のバリデーション(サーバーサイドバリデーションのような)を抑制する必要がある場合に便利です。(後ほど説明します)
```
Comment (hypermkt) [inline docs/ja/timing.md]: I'm worried about this translation a little bit. Is this accurate? hmm...

### original

> This is useful, when you need to suppress the validation (like the server-side validation) with async validation feature (explain later).

Comment (kazupon) [inline docs/ja/timing.md]: sorry, my poor english ... orz
日本語でいうと下記ですね。

```
これはサーバサイドバリデーションのように非同期にバリデーションする必要がある場合は便利です。
```

Comment (hypermkt) [inline docs/ja/timing.md]: oh, It sounds natural! Thanks for the comment!

**Comment by hypermkt** [comment]
Glad to back to stable. Why did it failed suddenly? hmm.

**Comment by kazupon** [comment]
Oh great!!
LGTM 👍

**Comment by hypermkt** [comment]
@kazupon 
I have finished translating! Could you review?

**Comment by kazupon** [comment]
published japanese translation docs
http://vuejs.github.io/vue-validator/ja/index.html

**Comment by hypermkt** [comment]
Ohhhh! It's released finally! Thank you!

**Pull request merged**